generate-release-notes.py

```diff
@@ -103,25 +140,27 @@ def enterprise_version():
             other.append(i)
 
     # Write release notes out to a file.
-    with io.open(FILENAME, "w", encoding='utf-8') as f:
-        f.write(u"%s\n\n" % date)
+    with io.open(FILENAME, "w", encoding="utf-8") as f:
```

Comment:
If we're changing this line anyway:

1. We can use `open()` instead of `io.open()`
2. We don't need to explicitly set encoding, since `utf-8` is the default